packages/starlight/translations/nl.json

```
@@ -0,0 +1,22 @@
+{
+  "skipLink.label": "Naar content",
```

Do you know if there’s a standard phrasing for skip links in Dutch pages? Had to try hard to find a Dutch site that actually had a skip link but [rijksoverheid.nl](https://www.rijksoverheid.nl/) uses “Ga direct naar inhoud” and [nldesignsystem.nl](https://nldesignsystem.nl/) uses “Ga naar hoofdinhoud”. (English Wikipedia has a “Jump to content” link, but Dutch doesn’t!)

Happy to stick with “content” if you think it’s better, but I wonder if “inhoud” would be more common?

```suggestion
  "skipLink.label": "Ga naar inhoud",
```

---

You're right! In traditional dutch, this would be a better fit!